umi_tools/group.py

```diff
@@ -403,7 +440,7 @@ def main(argv=None):
                         else:
                             # Add the 'UG' tag to the read
                             read.tags += [('UG', unique_id)]
-                            read.tags += [('FU', top_umi)]
+                            read.tags += [('BX', top_umi)]
```

Comment:
Hummm... I'm not really sure how works the BARCODE_TAG/READ_ONE_BARCODE_TAG/READ_TWO_BARCODE_TAG in Picard, but I think that the assigned UMI in PICARD is marked with MI:
```
ASSIGNED_UMI_TAG (String)	Tag name to use for assigned UMI Default value: MI. This option can be set to 'null' to clear the default value.
```
https://broadinstitute.github.io/picard/command-line-overview.html#UmiAwareMarkDuplicatesWithMateCigar